protocol/storage/miner.go

```diff
@@ -592,6 +603,7 @@ func (sm *Miner) OnCommitmentAddedToChain(sector *sectorbuilder.SealedSectorMeta
 		errMsg := fmt.Sprintf("failed sealing sector: %d", sectorID)
 		sm.dealsAwaitingSeal.fail(sector.SectorID, errMsg)
 	} else {
+		sm.dealsAwaitingSeal.addCommitMessageCid(sectorID, msgCid)
```

Comment:
SUGGESTION: Is there any circumstance where you'd want to add a commit message but the sector had not sealed successfully? If not, I think you should just add the commit message CID to the `sm.dealsAwaitingSeal.success` call and remove `addCommitMessageCid`.

Reply:
I had the same thought. It led to a bigger refactor (#2694), so I kept it out of this PR.